packages/discovery-provider/src/tasks/index_rewards_manager.py

```diff
@@ -132,9 +132,11 @@ def get_challenge_type_map(
             .all()
         )
         challenge_type_map_global = {
-            challenge[0]: TransactionType.trending_reward
-            if challenge[1] == ChallengeType.trending
-            else TransactionType.user_reward
+            challenge[0]: (
+                TransactionType.trending_reward
```

Comment:
formatting?

Reply:
yes, it was just the linter changing it (I ran `black .`)